cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -17,12 +17,669 @@
 Cask Data Application Platform Release Notes
 ============================================
 
+.. New Features
+.. Improvements
+.. Bug Fixes
+.. Known Issues
+.. API Changes
+.. Deprecated and Removed Features
+
 .. contents::
    :local:
    :class: faq
    :backlinks: none
    :depth: 2
 
+`Release 4.0.0 <http://docs.cask.co/cdap/4.0.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+#. Cask Market
+
+   - :cask-issue:`CDAP-7203` - Adds the new Cask Market view in the CDAP UI for users to
```

Comment:
Adds Cask Market: Cask’s Big data app store, that provides an ecosystem of pre-built hadoop solutions, re-usable templates, and plugins. Within CDAP users can access market and create hadoop solutions, big data applications with easy to use guided wizards.